### PR TITLE
Issue #254 Adding document for using OpenShift docker registry

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: docs/source/_topic_map.yml [hunk 1]
```diff
@@ -29,6 +29,8 @@ Topics:
         File: reusing-docker-daemon
       - Name: Troubleshooting Minishift
         File: troubleshooting
+      - Name: Accessing the OpenShift docker registry
```
Comment (budhrg) [inline docs/source/_topic_map.yml]: 👍

File: docs/source/developing/writing-docs.adoc [hunk 2]
```diff
@@ -17,6 +17,9 @@ documentation is a mix of generated
 https://en.wikipedia.org/wiki/Markdown[Markdown] files and manually
 maintained https://en.wikipedia.org/wiki/AsciiDoc[Asciidoc] files.
 
+If you are adding a new file to the doumentation please update
+ the `_topic_map.yml`  present in the root of `docs/source` sub-folder.
```
Comment (Preeticp) [inline docs/source/developing/writing-docs.adoc]: If you add a new file to the documentation...
you can remove the word 'present'

File: docs/source/using/accessing-the-openshift-docker-registry.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+[[openshift-docker-registry-intro]]
+== OpenShift Docker Registry
+
+OpenShift provides an integrated docker registry which can be used for local development as well.
+
+This helps the local development as the images present in registry can be used for applications running in OpenShift. Refer to the link:../using/accessing-the-openshift-docker-registry{outfilesuffix}#staging-app-with-openshift-registry[staging your application with the OpenShift registry] to know more.
```
Comment (Preeticp) [inline docs/source/using/accessing-the-openshift-docker-registry.adoc]: ...the registry...
Refer to the 'S'taging your application with the OpenShift registry section...

File: docs/source/using/accessing-the-openshift-docker-registry.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Note: Starting with OpenShift Origin release v1.5.0-alpha.2, the service IP of the registry is fixed at 172.30.1.1 and port 5000.
+
+[[login-to-openshift-docker-registry]]
+=== Login to the docker registry
+
+. Start minishift and add the `oc` binary to the PATH. Refer link:../getting-started/quickstart{outfilesuffix}[Minishift quickstart] for details.
+
+
+. Make sure youre shell is configured to  link:../using/reusing-docker-daemon{outfilesuffix}[reuse the Minishift docker daemon].
```
Comment (Preeticp) [inline docs/source/using/accessing-the-openshift-docker-registry.adoc]: Make sure your

File: docs/source/using/accessing-the-openshift-docker-registry.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+. Login to the docker registry
+
+ $ docker login -u developer -p `oc whoami -t` 172.30.1.1:5000
+
+[[staging-app-with-openshift-registry]]
+=== Staging your application with the OpenShift registry
+
+The following example shows how to deploy an OpenShift application directly from a locally built docker image.
+
+In this example OpenShift project "myproject" is used as it is automatically created and available for use after `minishift start`.
```
Comment (Preeticp) [inline docs/source/using/accessing-the-openshift-docker-registry.adoc]: comma after example

**Comment by hferentschik** [comment]
Merged with some amendments

**Pull request closed without merging**